src/app/containers/IndexPageSection/index.test.jsx

```diff
@@ -1,12 +1,10 @@
+import React from 'react';
```

Comment:
The way React was mocked here was creating issues with Emotion, and the mocking was unnecessary anyway